components/automate-cli/pkg/verifyserver/services/nfsmountservice/nfsmountservice.go

```diff
@@ -0,0 +1,260 @@
+package nfsmountservice
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+type INFSService interface {
+	GetNFSMountDetails(models.NFSMountRequest, bool) *[]models.NFSMountResponse
+}
+
+type NFSMountService struct {
+}
+
+func NewNFSMountService() INFSService {
```

Comment:
accept the log and port as method parameters and initialize them to the NFSMountService object